### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by rserafim** [vercel/next.js#41253] [Help]
I have this same error on 2 different sites. Initially I thought it was the old Safari version on the client's Iphone.
But it happened on another client's Chrome.

https://ibb.co/VtY4gJ9
https://ibb.co/bRhxh3W

urls: http://www.rscoelhotransportes.com.br / charlovillamassoterapia.com.br

I have no idea where to start.

{
  "private": true,
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "export": "npm run build && next export"
  },
  "dependencies": {
    "@splidejs/react-splide": "^0.7.8",
    "lightgallery": "^2.6.0",
    "markdown-it": "^13.0.1",
    "next": "latest",
    "react": "18.1.0",
    "react-dom": "18.1.0"
  },
  "devDependencies": {
    "@types/markdown-it": "^12.2.3",
    "@types/node": "^18.7.6",
    "@types/react": "^18.0.17",
    "@types/react-dom": "18.0.5",
    "autoprefixer": "^10.4.7",
    "eslint": "8.22.0",
    "eslint-config-next": "12.2.5",
    "frontmatter-markdown-loader": "^3.7.0",
    "postcss": "^8.4.14",
    "tailwindcss": "^3.1.2",
    "typescript": "4.7.2"
  }
}

**Comment by gaston-flores** [comment]
Unless you can get your client to give you some additional info or manage to reproduce it yourself, it'll be hard to figure out. Consider using an error reporting tool like [Sentry](https://sentry.io/welcome/) to catch these sort of errors.

**Reply by rserafim** [reply]
I added let's see if anything appears.

**Reply by HardikRathod123** [reply]
were you able to figure this out ? I am encountering similar issue where my web app is working on all devices except iPhone
